EIPS/eip-7719.md

```diff
@@ -0,0 +1,368 @@
+---
+eip: 7719
+title: P2P History Network
+description: Instead of each node storing a full copy of historical history divide the responsibility amongst the nodes on the network.
+author: Kolby ML (@KolbyML)
+discussions-to: https://ethereum-magicians.org/t/eip-7719-p2p-history-network/20231
+status: Draft
+type: Standards Track
+category: Networking
+created: 2024-06-05
+requires: 7643, 7718
+---
+
+## Abstract
+
+An overlay network over Discv5 based off the `Portal Wire Protocol` [EIP-7718](./eip-7718.md). This overlay network is called the Portal History Network and serves headers, block bodies, and receipts. Look ups on the network are done via block hash. All data has strong validity conditions, but proving path will differ based certain fork transitions
+
+## Motivation
+
+Historical blocks and receipts currently occupy more than 700GB of disk space (and growing!). Therefore, to validate the chain, users must typically have a 2TB disk.
+
+With lightweight sync strategies based on the PoS weak subjectivity assumption, which was possible after the merge. Storing all the block history was no longer required to sync and played a centralizing force increasing the storage requirements and sync times to participate in the Ethereum Network.
+
+Instead of each Ethereum node storing a full copy of the block history, what if this burden was divided amongst all nodes enabling each node to store a fraction of the blocks, but having access to all historical blocks as a collective lowering the cost for everybody, well maintaining the current JSON-RPC interfaces.
```

Comment:
```suggestion
Instead of each Ethereum node storing a full copy of the block history, what if this burden was divided amongst all nodes enabling each node to store a fraction of the blocks, but having access to all historical blocks as a collective lowering the cost for everybody, while maintaining the current JSON-RPC interfaces.
```